projects/ltc3208/Makefile

```diff
@@ -0,0 +1,11 @@
+EXAMPLE ?= dummy
```

Comment:
i really dont like `dummy` as example name, make it `basic` or `bringup` or whatever doesn't have that many meanings including negative ones